Clients/src/presentation/components/Table/RisksTable/index.tsx

```
@@ -0,0 +1,236 @@
+                <TableCell sx={cellStyle}>{row.risk_name}</TableCell>
+                <TableCell sx={cellStyle}>{row.risk_level}</TableCell>
+                <TableCell sx={cellStyle}>{row.owner}</TableCell>
+                <TableCell sx={cellStyle}>
+                  {row.review_date
+                    ? formatDate(row.review_date.toString())
+                    : "No review date"}
+                </TableCell>
+                <TableCell sx={singleTheme.tableStyles.primary.body.cell}>
+                </TableCell>
+              </TableRow>
+            ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)